advisor/build.gradle.kts

```diff
@@ -26,6 +27,7 @@ plugins {
 
 dependencies {
     api(project(":clients:nexus-iq"))
+    api(project(":clients:oss-index"))
```

Comment:
I'm surprised to see that you've started a separate client project in ORT for this instead of using our very own [ossindex-service-client](https://github.com/sonatype/ossindex-public). Could you please elaborate the details of this decision?

I actually had planned to use *ossindex-public* myself, but ran into some limitations. I started to [address one of them](ossindex-public), but then there were some CLA issues on the Sonetype side. So @DarthHater, maybe you could first help us to resolve these:

https://github.com/sonatype/ossindex-public/issues/7
https://github.com/sonatype/ossindex-public/issues/17
https://github.com/sonatype/ossindex-public/issues/24

Reply:
Yeah I mostly actually did this out of expedience to learn how the project worked, and Kotlin. I can see if I can help unstick those!